src/index.html

```diff
@@ -3,12 +3,36 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+          content="width=device-width,
+          user-scalable=no,
+          initial-scale=1.0,">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:500&display=swap " rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header ">
+      <a href="index.html" class="header__logo">
+        <img src="images/logo.png" alt="logo">
+      </a>
+      <nav class = "nav">
+        <ul class="nav__list">
+          <li><a href="#" class = "nav__link is-active">Apple</a></li>
+          <li><a href="#" class = "nav__link">Samsung</a></li>
+          <li><a href="#" class = "nav__link">Smartphones</a></li>
```

Comment:
```suggestion
          <li>
            <a href="#" class = "nav__link is-active">Apple</a>
          </li>
          <li>
            <a href="#" class = "nav__link">Samsung</a>
          </li>
          <li>
            <a href="#" class = "nav__link">Smartphones</a>
          </li>
```
it will increase readibility. make it for all links